client/src/services/assignments.js

```diff
@@ -0,0 +1,80 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+
+// SPDX-License-Identifier: MIT
+
+// The parameter 'indices' used in the following functions is an array of integres 
```

Comment:
You've made the indices work so the code here seems good to me. If any problems occur with the mapping to parent assignments, would consider ids or something. But no immediate changes needed imo.

Reply:
Good idea, I'll probably look into this while implementing the edit assignment view and see if this will be an easy change to make.